charts/openebs/v1.11.0/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
-version: 1.10.0
+version: 1.11.0
 name: openebs
-appVersion: 1.10.0
+appVersion: 1.11.0
 description: Containerized Storage for Containers
 icon: https://raw.githubusercontent.com/cncf/artwork/master/projects/openebs/icon/color/openebs-icon-color.png
```

Comment:
Please make a local copy of the logo in the parent folder `openebs/` and reference the image here as `file://../openebs-icon-color.png`. This way the logo can be displayed in airtight environments.

Reply:
Changed as per suggestion. Please take a look.